File: Cite.EvalIt/Query/DataObjectTypeLookup.cs
using Cite.EvalIt.Common;
using Cite.Tools.Data.Query;
using System;
using System.Collections.Generic;

namespace Cite.EvalIt.Query
{
	public class DataObjectTypeLookup : Lookup
	{
		public string Like { get; set; }
		public List<Guid> Ids { get; set; }
		public List<Guid> ExcludedIds { get; set; }
		public List<IsActive> IsActive { get; set; }
	}
}

File: Cite.EvalIt.WebApp/src/app/core/query/data-object-type.lookup.ts
import { IsActive } from '@app/core/enum/is-active.enum';
import { Lookup } from '@common/model/lookup';
import { Guid } from '@common/types/guid';

export class DataObjectTypeLookup extends Lookup implements DataObjectTypeFilter {
	ids: Guid[];
	excludedIds: Guid[];
	isActive: IsActive[];
	like: string;

	constructor() {
		super();
	}
}

export interface DataObjectTypeFilter {
	ids: Guid[];
	excludedIds: Guid[];
	isActive: IsActive[];
	like: string;
}